### ISSUE TITLE
Many view iterators do not contain an `_Unwrapped()` member function

### ISSUE THREAD

**Issue by strega-nil-ms** [microsoft/STL#2997]
While implementing tests for #2991, I noticed that view iterators are always `_Is_nothrow_unwrappable_v`; this is because `_Get_unwrap()` does no work on them, as they don't have `_Unwrapped()` member functions.

It would be good to add `_Unwrapped()` member functions which return the same iterator, but using the unwrapped base.

**Comment by CaseyCarter** [comment]
> It would be good to add `_Unwrapped()` member functions which return the same iterator, but use the unwrapped base.

Of course, this is only applicable when a range of adapted iterators corresponds exactly to a range of underlying iterators, i.e., `drop_view` or `take_view` iterators could always be unwrapped but not `filter_view` or `stride_view` iterators.
